python/tk_multi_loader/dialog.py

```diff
@@ -1180,16 +1201,26 @@ def _add_rdo_list_view_mod(self):
             self.ui.label_2.hide()
             self.ui.thumb_scale.hide()
 
-    def _add_rdo_publish_search(self):
+    def _add_rdo_status_filter(self, hlayout):
+        '''
+        Add a combo box to sort the latest publishes by status
+        Kind of copying off the _add_rdo_status_filter
+        '''
+        filter_label = QtGui.QLabel("Filter by Status")
```

Comment:
Instead of a bunch of spaces you should edit the stylesheet a little bit. Check https://bitbucket.org/rodeofx/tk-framework-rdo/src/f2827d220ca5453a086294a35ccdbb4bf3b8356a/v0.0.1/python/dialogs/LoadMultiple.py?at=master#LoadMultiple.py-97